buildSrc/build.gradle.kts

```diff
@@ -0,0 +1,35 @@
+plugins {
+    java
+    `kotlin-dsl`
+}
+
+repositories {
+    mavenCentral()
+    gradlePluginPortal()
+}
+
+/**
+ * The version of the Kotlin Gradle plugin.
+ */
+val kotlinVersion = "1.4.20"
+
+configurations.all {
+    resolutionStrategy {
+        force(
+            // Force Kotlin lib versions avoiding using those bundled with Gradle.
+            "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion",
+            "org.jetbrains.kotlin:kotlin-stdlib-common:$kotlinVersion",
+            "org.jetbrains.kotlin:kotlin-reflect:$kotlinVersion"
+        )
+    }
+}
+
+val jvmVersion = JavaLanguageVersion.of(17)
```

Comment:
Please have a comment on the fact that this value should be kept consistent with the value defined in `BuildSettings.kt`.